librad/src/identities/git.rs

```diff
@@ -379,6 +410,24 @@ where
     }
 }
 
+impl<'a, T: 'a> Identities<'a, VerifiedIdentity<T>> {
+    #[tracing::instrument(level = "debug", skip(self, mine, theirs), err)]
```

Comment:
Suggestion: use `fields()` in the `instrument` macro to establish the `content_id`s / `revision`s in the span scope, then log just the strings.